src/utils/hooks/useAvatarDisplayCounter.ts

```diff
@@ -1,9 +1,8 @@
-import Maybe from 'graphql/tsutils/Maybe';
 import { useMemo } from 'react';
 
 const useAvatarDisplayCounter = (
   maxAvatars: number,
-  members: Maybe<string[]>,
+  members: any[],
```

Comment:
Can we use ColonyContributor[] | ColonyWatcher[] | undefined instead of any?

Reply:
:point_up:

Reply:
Yeah, it was a placeholder that I forgot to fix :)